JitsiConference.js

```diff
@@ -2086,9 +2087,16 @@ JitsiConference.prototype._onIceConnectionFailed = function(session) {
         // and "bad" connection
         Statistics.analytics.addPermanentProperties({ p2pFailed: true });
 
-        // Log analytics event, but only for the initiator to not count it twice
-        if (this.p2pJingleSession && this.p2pJingleSession.isInitiator) {
-            Statistics.sendEventToAll(P2P_FAILED);
+        if (this.p2pJingleSession) {
+            const event
+                = createP2pEvent(
+                    'failed',
+                    {
+                        initiator: this.p2pJingleSession.isInitiator
+                    });
+
+            Statistics.sendLog(JSON.stringify(event));
+            Statistics.analytics.sendEvent(event);
```

Comment:
I noticed that the call to sendEventToAll was removed in some cases but not others. What were your requirements for using sendEventToAll versus not using it?

Reply:
sendEventToAll only works with the analytics.sendEvent(eventName, properties) API. If I have an event object instead I can't use it, so I split the call. I'll add another "send-to-all" kind of function as Brian suggested.